omnibus/files/server-ctl-cookbooks/infra-server/recipes/default.rb

```diff
@@ -148,7 +148,11 @@
       action :disable
     end
   elsif node['private_chef'][service]['enable']
-    include_recipe "infra-server::#{service}"
+    if service == 'elasticsearch' && node['private_chef']['opscode-erchef']['search_provider'] == 'opensearch' && node['private_chef']['opensearch']['external']
```

Comment:
Preflight check: If the search_provider is opensearch & opensearch external is not enabled then raise it in preflight checks

Reply:
Add opensearch-external.rb

Reply:
This still seems off to me. I know we talked about it, but could you refresh my memory - why can't we stick to the basic "infra-server::#{service}" and "infra-server::#{service-external}" behavior here?